queries/omen/markets.ts

```diff
@@ -155,10 +156,11 @@ const getMarketsQuery = (
         ${params.category_contains ? 'category_contains: $category_contains' : ''}
         ${params.openingTimestamp_gt ? 'openingTimestamp_gt: $openingTimestamp_gt' : ''}
         ${params.openingTimestamp_lt ? 'openingTimestamp_lt: $openingTimestamp_lt' : ''}
-        ${params.isPendingArbitration ? 'isPendingArbitration: $isPendingArbitration' : ''}
-        ${params.currentAnswer ? 'currentAnswer: $currentAnswer' : ''}
+        ${params.isPendingArbitration !== undefined ? 'isPendingArbitration: $isPendingArbitration' : ''}
+        ${params.currentAnswer !== undefined ? 'currentAnswer: $currentAnswer' : ''}
```

Comment:
Same here, if filtered for `Pending`, then `currentANswer` is `null`, which results to not including it in filters and then, markets that are already resolved are shown

Reply:
Yes, this too can be included.